packages/firebase_installations/firebase_installations/ios/Classes/IdChangedStreamHandler.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
```

Comment:
License headers on all ios files please, and check other new files like dart ones as well if possible, thanks :)